scripts/runSFCCJob.js

```diff
@@ -0,0 +1,80 @@
+// scripts/runSFCCJob.js
+require('dotenv').config();
+const sfcc = require('sfcc-ci');
+const authenticate = require('./auth');
+
+(async () => {
+    try {
+        const token = await authenticate();
+
+        const instance = process.env.SANDBOX_HOST;
+        const jobId = 'AlgoliaProductIndex_v2';
+
+        // Run Job
+        const jobExecution = await new Promise((resolve, reject) => {
+            sfcc.job.run(instance, jobId, {}, token, (err, result) => {
+                if (err) {
+                    console.error('Job run error:', err);
+                    console.error('Error details:', JSON.stringify(err, null, 2));
+                    reject(err);
+                } else {
+                    console.log('Job started, result:', JSON.stringify(result, null, 2));
+                    resolve(result);
+                }
+            });
+        });
+
+        console.log('Job execution object:', JSON.stringify(jobExecution, null, 2));
+
+        const jobExecutionId = jobExecution.body.id;
+        if (!jobExecutionId) {
+            console.error('Job execution details:', JSON.stringify(jobExecution, null, 2));
+            throw new Error('Failed to get job execution ID');
+        }
+
+        // Poll for Job Completion
+        let attempts = 0;
+        const maxAttempts = 15;
+        const delay = 30000; // 30 seconds
```

Comment:
Optim: as the job takes between 1 and 2 min, waiting 30s is a bit too much.
```suggestion
        const delay = 15000; // 15 seconds
```